.travis.yml

```diff
@@ -84,6 +84,7 @@ script:
 - JSONPATH='{range .items[*]}{@.metadata.name}:{range @.status.conditions[*]}{@.type}={@.status};{end}{end}'; until kubectl -n kube-system get pods -lk8s-app=kube-dns -o jsonpath="$JSONPATH" 2>&1 | grep -q "Ready=True"; do sleep 1;echo "waiting for kube-dns to be available"; kubectl get pods --all-namespaces; done
 - helm install -f helm-configs/.travis-galaxy-sc-deployment.yaml --name travis --version $HELM_REPO_VERSION galaxy-helm-repo/galaxy-stable
 - planemo lint -r --report_level error tools
+- pushd tools/tertiary-analysis/scanpy > /dev/null && planemo t && popd > /dev/null  
```

Comment:
Maybe we can make this as a for for the different directories where we have tests, and then it is easier to add that there once we have more tools with tests? When it comes to executing the planemo tests, this is a setup I used recently somewhere else:

https://github.com/AstraZeneca-Omics/immport-galaxy-tools/blob/master/.travis.yml#L32

Reply:
For the testing there I had to fix the python version to avoid some issues.

Reply:
But then logic in the link above will avoid running tests if there are no changes there.

Reply:
Thanks @pcm32 - I'm trying to implement that logic, but it's not running any tests. Would you have any idea why?

Reply:
hmm, it's an issue with the Git range